static/js/connections.js

```diff
@@ -19,7 +19,7 @@ Table.renderTable('table', tableOptions, function (tr, item, all) {
   if (all) {
     const connectionLink = document.createElement('a')
     connectionLink.href = `connection#name=${encodeURIComponent(item.name)}`
-    if (item.client_properties.connection_name) {
+    if (item.protocol !== 'MQTT' && item.client_properties.connection_name) {
```

Comment:
I think we should avoid conditions on protocol. Instead check if properties are set or not and similar. Is it that `client_properties` are null or empty here? If so, maybe just do something like
```suggestion
    if (item?.client_properties?.connection_name) {
```

Reply:
ok yes i agree. nice, thanks!